Quadriga/src/main/webapp/WEB-INF/views/public/searchsites.jsp

```diff
@@ -0,0 +1,66 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+	pageEncoding="ISO-8859-1"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ taglib uri="http://www.springframework.org/tags" prefix="spring"%>
+
+<script>
+	$(document).ready(function() {
+		$('#example1').DataTable({
+			"searching" : false
+		});
+	});
+
+	function handle(e) {
+		if (e.keyCode === 13) {
+			document.getElementById("sites1").submit();
+		}
+		return false;
+	}
+
+	function callMethod() {
+		document.getElementById("sites1").submit();
+	}
+</script>
+
+<form name='searchsites' id="sites1"
+	action="${pageContext.servletContext.contextPath}/sites/searchTerm"
+	method='POST'>
+	<table width="100%">
+		<tr>
+			<td style="width: 85%"><input type="text" class="form-control"
+				placeholder="Search Public Sites" name="searchTerm" tabindex="1"
+				onkeypress="handle(event)" value="${searchTerm}"
+				autocapitalize="off"></td>
+			<td style="width: 2%"></td>
+			<td style="width: 25%"><input type="submit" value="Search"
+				style="width: 100%; height: 115%; font-weight: bold;" tabindex="2"
+				onclick="callMethod()"></td>
```

Comment:
why do you use javascript to submit the form, if you don't do anything else then submitting it? am I missing something?